src/Qowaiv/DecimalRound.cs

```diff
@@ -4,81 +4,11 @@
 namespace Qowaiv
 {
     /// <summary>Extensions on <see cref="decimal"/> rounding.</summary>
-    public static class DecimalRound
+    internal static class DecimalRound
     {
         private const int ScaleMask = 0x00FF0000;
         private const int SignMask = unchecked((int)0x80000000);
 
-        /// <summary>Returns true if the rounding is direct; the nearest of the two options is not relevant.</summary>
-        [Pure]
-        public static bool IsDirectRounding(this DecimalRounding mode)
-            => mode >= DecimalRounding.Truncate && mode <= DecimalRounding.Floor;
-
-        /// <summary>Returns true if rounding is to the nearest. These modes have half-way tie-breaking rule.</summary>
-        [Pure]
-        public static bool IsNearestRounding(this DecimalRounding mode)
-            => mode >= DecimalRounding.ToEven && mode <= DecimalRounding.RandomTieBreaking;
-
-        /// <summary>Rounds a value to the closed number that is a multiple of the specified factor.</summary>
-        /// <param name="value">
-        /// The value to round to.
-        /// </param>
-        /// <param name="multipleOf">
-        /// The factor of which the number should be multiple of.
-        /// </param>
-        /// <returns>
-        /// A rounded number that is multiple to the specified factor.
-        /// </returns>
-        [Pure]
-        public static decimal RoundToMultiple(this decimal value, decimal multipleOf) => value.RoundToMultiple(multipleOf, DecimalRounding.ToEven);
-
-        /// <summary>Rounds a value to the closed number that is a multiple of the specified factor.</summary>
-        /// <param name="value">
-        /// The value to round to.
-        /// </param>
-        /// <param name="multipleOf">
-        /// The factor of which the number should be multiple of.
-        /// </param>
-        /// <param name="mode">
-        /// The rounding method used to determine the closed by number.
-        /// </param>
-        /// <returns>
-        /// A rounded number that is multiple to the specified factor.
-        /// </returns>
-        [Pure]
-        public static decimal RoundToMultiple(this decimal value, decimal multipleOf, DecimalRounding mode)
```

Comment:
RoundToMultiple not needed anymore?

Reply:
Exposed by extensions. So only the root method is needed that is exposed via extensions too.